src/js/chartAuthoring.js

```diff
@@ -66,6 +68,12 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                                 },
                                 styles: {
                                     text: "floe-inlineEdit-text"
+                                },
+                                listeners: {
+                                    "afterFinishEdit": {
```

Comment:
The listener can be simplified to:
```
"afterFinishEdit.modelChange": {
    func: "{pieChart}.applier.change",
    args: ["pieTitle", "{arguments}.0"]
}
```
Same comment applies to the listener at line 95